### DISCUSSION TITLE
Uncaught (in promise) SyntaxError: JSON.parse: unexpected non-whitespace character after JSON data at line 1 column 542 of the JSON data

### DISCUSSION THREAD

**Discussion by ilahazs** [livewire/livewire#7208] [1. Help]
I've a problem on my site that using Livewire 3, I almost not code on that app in 2 weeks, and after i re-check it there's an error on console said:
Uncaught (in promise) SyntaxError: JSON.parse: unexpected non-whitespace character after JSON data at line 1 column 542 of the JSON data

how to fix this? my domain is https://rean.id/login

**Comment by LAGGOUNE-Walid** [comment]
its the same error for me ! 
https://github.com/livewire/livewire/discussions/7280

have you found any solution ?

**Reply by ilahazs** [reply]
Yes it's fixed for me. What I did is reinstall php8.2-fpm and change the port of php-fpm to another port.

**Reply by LAGGOUNE-Walid** [reply]
Okay i will try it now , i changed my php version but didn't work , but i will change the port .

**Reply by ilahazs** [reply]
Sorry I can't explain it properly, but in my case, the previous PHP-FPM version that installed on my server was 8.1. After changing it to 8.2, and serving it on another port, and restarting Nginx, the issue seems to be resolved

**Reply by LAGGOUNE-Walid** [reply]
I changed my fpm to 8.2 and i changed the port but nothing changed . the error do not appears in first requests but after few tries it appears again . but thank you for your help.

**Reply by ilahazs** [reply]
Sorry to hear that, I hope you find the solution of it, but pretty sure the problem is came from php-fpm otherwise is came from VM server.

**Reply by LAGGOUNE-Walid** [reply]
Thanks, I will continue trying to change the docker php and nginx  versions

**Reply by ilahazs** [reply]
Hi, it's been a weeks now. Does the problem already fixed?

**Reply by LAGGOUNE-Walid** [reply]
Hello , thanks for asking again . yes i think now everything is ok . the one thing i did is i closed the php 9000 port in docker-compose.yml . until now i don't know why and how this could be the problem . but everything works fine now . thank you again .

**Reply by ilahazs** [reply]
Glad to hear it bro!

**Reply by ilzicode** [reply]
> Hello , thanks for asking again . yes i think now everything is ok . the one thing i did is i closed the php 9000 port in docker-compose.yml . until now i don't know why and how this could be the problem . but everything works fine now . thank you again .

thanks a lot.. i have the same problem.. and it solve by change the port 9000 in to 9001..

**Comment by Henriquex25** [comment]
Did you find any solution? I have the same problem

**Reply by LAGGOUNE-Walid** [reply]
Hi there , there is no direct and clear solution , as i said before i just closed the php 9000 docker port . i hope this will work for you .

**Reply by wallacemaxters** [reply]
In some cases I was able to resolve this by adding `pm=ondemand` in the `php-fpm.conf` file

**Reply by Henriquex25** [reply]
I changed it to port 9001 in Docker Compose and it seems to have resolved it. So far the error has not occurred

**Comment by aliqasemzadeh** [comment]
```
Uncaught (in promise) SyntaxError: Unexpected non-whitespace character after JSON at position 579 (line 1 column 580)
    at JSON.parse (<anonymous>)
    ```
    
    I have this problem too.
